packages/model/CsvColumn.ts

```diff
@@ -4,12 +4,15 @@ import { Constructor, property, ResourceIdentifier } from '@tpluscode/rdfine'
 import type { GraphPointer } from 'clownface'
 import { csvw, dtype, schema } from '@tpluscode/rdf-ns-builders'
 import { cc } from '@cube-creator/core/namespace'
+import { NamedNode } from '@rdfjs/types'
 import { initializer } from './lib/initializer'
+import { inferDatatype } from './lib/datatypeInference'
 
 export interface CsvColumn extends RdfResource {
   name: string
   order: number
   samples: string[]
+  defaultDatatype: NamedNode
```

Comment:
maybe a better name would be `guessedDatatype`, or `inferredDatatype` to align with the module name?

Reply:
the subtle rationale for choosing the name would be that the inferred datatype is used as a default value. But this doesn't matter anymore since, as you pointed out in the next comment, there's no need to carry this value around as a property of `CsVColumn`